Readme.md

```diff
@@ -0,0 +1,147 @@
+README.md
+One attachment
+• Scanned by Gmail
```

Comment:
- [ OPTIONAL ] Great job making the changes requested by the previous reviewers :+1:  Please consider removing this block of text as it is of no relevance to your README file. Doing this can help keep your repository professional :100: